certified-connectors/BttnV2/Readme.md

```diff
@@ -0,0 +1,18 @@
+## BttnV2 Connector
+The magical bttn that automates your business, improves your customer experience, and so much more…   
+This connector is for the bttn ONE. The next generation of Internet of Things buttons.
+
```

Comment:
@WesselBruijnes Please refer the template for readme.md file and add required sections.